cve_bin_tool/checkers/libjpeg.py

```diff
@@ -5,7 +5,7 @@
 CVE checker for libjpg-turbo
 
 Note that this file is named libjpeg.py instead of libjpeg-turbo.py to avoid an issue
-with loading the module.
+with loading the product.
```

Comment:
```suggestion
with loading the checker.
```

We're not loading libjpeg.so (the product), we're loading the checker.